### PR TITLE
feat(DMS): rocketmq supports public access updated

### PR DESCRIPTION
**Pull request by 346010762**
**What this PR does / why we need it**:
 rocketmq supports to enable or disable publicip

**Special notes for your reviewer**:

**Release note**:

```release-note
rocketmq supports to enable or disable publicip 
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/s/acceptance/dms/ TESTARGS='-run TestAccDmsRocketMQInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDmsRocketMQInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccDmsRocketMQInstance_basic
=== PAUSE TestAccDmsRocketMQInstance_basic
=== CONT  TestAccDmsRocketMQInstance_basic
--- PASS: TestAccDmsRocketMQInstance_basic (775.66s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       775.699s

make testacc TEST=./huaweicloud/services/acceptance/dms/ TESTARGS='-run TestAccDmsRocketMQInstance_prepaid_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDmsRocketMQInstance_prepaid_basic -timeout 360m -parallel 4
=== RUN   TestAccDmsRocketMQInstance_prepaid_basic
=== PAUSE TestAccDmsRocketMQInstance_prepaid_basic
=== CONT  TestAccDmsRocketMQInstance_prepaid_basic
--- PASS: TestAccDmsRocketMQInstance_prepaid_basic (897.69s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       897.739s

make testacc TEST=./huaweicloud/services/acceptance/dms/ TESTARGS='-run TestAccDmsRocketMQInstance_publicip'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDmsRocketMQInstance_publicip -timeout 360m -parallel 4
=== RUN   TestAccDmsRocketMQInstance_publicip
=== PAUSE TestAccDmsRocketMQInstance_publicip
=== CONT  TestAccDmsRocketMQInstance_publicip
--- PASS: TestAccDmsRocketMQInstance_publicip (670.66s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       670.708s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go [hunk 1]
```diff
@@ -293,3 +339,85 @@ resource "huaweicloud_dms_rocketmq_instance" "test" {
...
+	return fmt.Sprintf(`
+%s
+
+data "huaweicloud_availability_zones" "test" {}
+
+data "huaweicloud_dms_rocketmq_flavors" "test" {
+  type = "single.basic"
+}
+
+resource "huaweicloud_vpc_eip" "eip1" {
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go]:
```suggestion
resource "huaweicloud_vpc_eip" "test" {
```

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go [hunk 2]
```diff
@@ -293,3 +339,85 @@ resource "huaweicloud_dms_rocketmq_instance" "test" {
...
+  publicip {
+    type = "5_bgp"
+  }
+
+  bandwidth {
+    name        = "test_eip1"
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go]:
```suggestion
    name        = "%s"
```

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go (hunk 1 above)
Comment (346010762) [reply huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go]: done

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go (hunk 2 above)
Comment (346010762) [reply huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go]: done

**Pull request merged**